### PR TITLE
GIVCAMP-180 | Scrollytelling POC

### PR DESCRIPTION
**Pull request by yvonnetangsu**
# READY FOR REVIEW

# Summary
- NYT style scrollytelling story POC

# Review By (Date)
- Retro

# Review Tasks

## Setup tasks and/or behavior to test

1. Look at https://deploy-preview-107--giving-campaign.netlify.app/stories/immersive-featured-story-test

# Associated Issues and/or People
- JIRA ticket(s) - GIVCAMP-107

### CODE REVIEW

File: components/Grid/Grid.tsx [hunk 1]
```diff
@@ -50,7 +50,7 @@ export const Grid = ({
   <AsComponent
     {...props}
     className={cnb(
-      'grid overflow-hidden',
```
Comment (yvonnetangsu) [inline components/Grid/Grid.tsx:53]: For `position: sticky` to work, the ancestor containers can't have `overflow:hidden`

**Comment by yvonnetangsu** [comment]
This is a POC which means it's cowboy day 🤠

**Comment by sherakama** [comment]
These gifs make me so happy.

**Comment by yvonnetangsu** [comment]
> ###
> [Aug 14, 2023](#issuecomment-1678235852)

Being able to add GIFs was a quality of life improvement for me 😂

**Pull request merged**